migrations/20210305124354_create_initial_tables.up.sql

```diff
@@ -0,0 +1,80 @@
+CREATE TABLE users
+(
+    id bigint NOT NULL,
+    first_name character varying(50),
+    last_name character varying(50),
+    email character varying(255) NOT NULL,
+    email_verified_at date,
+    created_at date,
+    updated_at date,
+    PRIMARY KEY (id),
+    CONSTRAINT users_email_uniq UNIQUE (email)
+);
+
+CREATE TABLE social_accounts
+(
+    id bigint NOT NULL,
+    user_id bigint NOT NULL,
+    provider character varying(30) NOT NULL,
+    provider_user_id bigint NOT NULL,
+    access_token character varying(255) NOT NULL,
+    refresh_token character varying(255),
+    expires_in bigint,
+    created_at date,
+    updated_at date,
+    PRIMARY KEY (id),
+    CONSTRAINT social_accounts_user_id_foreign FOREIGN KEY (user_id)
+        REFERENCES users (id) MATCH SIMPLE
+        ON UPDATE CASCADE
+        ON DELETE CASCADE
+        NOT VALID
+);
```

Comment:
i'm not sure we actually need this... do we?

Reply:
My thought was supporting multiple auth providers but if we're only ever going to use Slack and we use the token to post to the Slack group then I agree.

Reply:
I looked at the Slack API some more and it doesn't look like we need a user token at all even if we want to support posting the job automatically as that can be done either as the user (requiring the user token) or the bot.  I looked and it looks like NashDev is using the "bot" token... thoughts?

Reply:
nice! i'm good with using the "bot" token

Reply:
i don't see us supporting multiple auth providers

Reply:
Yea I started to adjust this but just haven't had time to circle back.  My original plan is to remove the social_accounts table and add some of the Slack details (user_id, access_token, team) to the users table.  For the current plan I'm not sure any of those are strictly necessary, though.  Also to switch to using Postgres's `SERIAL` or `BIGSERIAL`.